package/gluon-web-logging/luasrc/lib/gluon/web/model/admin/logging.lua

```diff
@@ -0,0 +1,49 @@
+local uci = require('simple-uci').cursor()
+local system = uci:get_first('system', 'system')
+
+local f = Form(translate('Logging'), translate(
+	'If you want to use a remote syslog server, you can set it up here. '
+	.. 'Please keep in mind, that the data is transmitted unencrypted.'
+))
+local s = f:section(Section)
+
+local enable = s:option(Flag, 'log_remote', translate('Enable'))
+enable.default = uci:get_bool('system', system, 'log_remote')
+enable.optional = true
+function enable:write(data)
+	uci:set('system', system, 'log_remote', data)
+end
+
+local ip = s:option(Value, 'log_ip', translate('IP'))
+ip.default = uci:get('system', system, 'log_ip')
+ip:depends(enable, true)
+ip.optional = false
+ip.placeholder = '0.0.0.0'
+ip.datatype = 'ipaddr'
+function ip:write(data)
+	if data ~= nil then
```

Comment:
You set *optional* to false, so *data* can never be nil. (And if it could be nil, `uci:set('system', system, 'log_ip', data)` would delete it with simple-uci, so the if/else would not be necessary anyways.)